content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,74 +1,355 @@
+
+.. seealso::
+   - :doc:`../../fiscal_localizations`
+
+For AvaTax this setting needs to be set to either :guilabel:`United States` or :guilabel:`Canada`.
+
+Company settings
+----------------
+
+All companies operating under the Odoo database should have a full and complete address listed in
+the settings. Navigate to :menuselection:`Settings App --> Companies`. Should there only be one
+company operating the Odoo database, click :guilabel:`Update Info` to update the reliant contact
+information for the company.
+
+Should there be multiple companies operating in the database, click :guilabel:`Manage Companies` to
+load a list of companies to select from. Update customer information by clicking into the specific
+customer.
+
+Database administrators should ensure that the :guilabel:`Street...`, :guilabel:`Street2...`,
+:guilabel:`City`, :guilabel:`State`, :guilabel:`ZIP`, and :guilabel:`Country` are all updated for
+the companies.
+
+Module installation
+-------------------
+
+Next, ensure that the Odoo *AvaTax* module is installed. To do so, navigate to :menuselection:`Apps
+application --> Search...` bar and then type in `avatax`. Press enter and the following results will
+populate:
+
+- :guilabel:`Avatax`: `account_avatax`
+- :guilabel:`Avatax for SO`: `account_avatax_sale`
+- :guilabel:`Avatax for Subscriptions`: `account_avatax_sale_subscription`
+- :guilabel:`Avatax Brazil`: `l10n_br_avatax`
+- :guilabel:`Avatax for SOs in Brazil`: `l10n_br_avatax_sale`
+- :guilabel:`Account Avatax - Ecommerce`: `website_sale_account_avatax`
+- :guilabel:`Account AvaTax - Ecommerce - Delivery`: `website_sale_delivery_avatax`
+
+Click on the :guilabel:`Install` button on the module labeled :guilabel:`Avatax`: `account_avatax`.
+This will install the following modules:
+
+- :guilabel:`Avatax`: `account_avatax`
+- :guilabel:`Avatax for SO`: `account_avatax_sale`
+- :guilabel:`Account Avatax - Ecommerce`: `website_sale_account_avatax`
+
+Should *AvaTax* be needed for Odoo *Subscriptions* or for delivery tax in Odoo *Ecommerce*, then
+install those modules individually by clicking on :guilabel:`Install`. Additionally, in Brazil two
+separate *AvaTax* modules need to be installed. See the list above for these modules.
+
+.. _avatax/credentials:
+
+Odoo AvaTax settings
+====================
+
+To integrate the *AvaTax* :abbr:`API (application programming interface)` with Odoo, go to
+:menuselection:`Accounting --> Configuration --> Settings --> Taxes --> AvaTax` section. This is
+where the *AvaTax* some configurations will be made and the credentials will be entered in.
 
 .. image:: avatax/avatax-configuration-settings.png
    :align: center
    :alt: Configure Avatax settings
 
-.. _avatax/tax-mapping:
+Prerequisites
+-------------
 
-Tax mapping
-===========
+Before the credentials are entered in, tick the checkbox to the right of the :guilabel:`AvaTax`
+title to activate the feature.
```

Comment:
I believe the checkbox to the right is for enabling another feature?

![image](https://github.com/odoo/documentation/assets/23611711/35621655-210e-4536-97c8-b93140685121)

Reply:
Good catch